docs/README.md

```diff
@@ -180,7 +178,7 @@ can contain includes of some parts from other files. Cyclic references are not
 permitted
 
 First includes resolved and then variables are processed, which means:
-- included files can contain variables which will be processed from same context
+- included files can contain variables which will be processed from the same context
 - include reference can NOT contain variable, following is invalid:
```

Comment:
-> `an` include reference can NOT contain `any` variable`s`. The following is invalid: